indexer/httpd/src/graphql/types/block.rs

```diff
@@ -56,18 +56,9 @@ pub struct BlockInfo {
 
 impl From<grug_types::BlockInfo> for BlockInfo {
     fn from(item: grug_types::BlockInfo) -> Self {
-        let epoch_millis = item.timestamp.into_millis();
-        let seconds = (epoch_millis / 1_000) as i64;
-        let nanoseconds = ((epoch_millis % 1_000) * 1_000_000) as u32;
-
-        let timestamp = sea_orm::sqlx::types::chrono::Utc
-            .timestamp_opt(seconds, nanoseconds)
-            .single()
-            .unwrap_or_default();
-
         Self {
             block_height: item.height,
-            timestamp,
+            timestamp: item.timestamp.to_utc_date_time(),
```

Comment:
@penso You use `NaiveDateTime` everywhere else but only `DateTime<Utc>` here. Is this intended?

Reply:
I'd say that's a miss on my side.

Reply:
Can you make a PR to use `NaiveDateTime` everywhere?